docs/release_notes.md

```diff
@@ -1,5 +1,147 @@
 #
 
+## 4.3.1
+
+Release Date: 2023-10-xx
+
+This release represents a steady improvement on 4.3.0's significant release last year.  Most significantly, the memory leaks introduced with the new frameworks in 4.3.0 have been fixed alongside internal refactoring.  Additionally, three new operating systems are now supported by our binary packaging.
+
+Detached mode has been added to the unixfilesystem plugin which provides superior performance when working with parallel filesystems.
+
+Additional machinery has been made available to control TCP Keepalive behavior, ticket administration, connection pooling, and authentication configuration.
+
+This release consists of [402 commits from 16 contributors](https://github.com/irods/irods/compare/4.3.0...4.3.1) and [closed 235 issues marked for 4.3.1](https://github.com/irods/irods/issues?q=milestone%3A4.3.1) and [an additional 119 closed issues included in the recent 4.2.12 release](https://github.com/irods/irods/issues?q=milestone%3A4.2.12+closed%3A%3E2022-06-12).
+
+The latest binary packages for CentOS7, AlmaLinux8, RockyLinux9, Ubuntu18, Ubuntu20, Ubuntu22, Debian11, and Debian12 are available at <https://packages.irods.org/>.
+
+The release notes include:
+
+### Enhancements
+
+ - Add TCP keepalive options configuration [#2533] [#3824] [#3986]
+ - Prompt for auth scheme and SSL information with iinit [#3164] [#6398]
+ - Include resource 'comment' and 'info' in izonereport [#3739]
+ - Add detached mode to unixfilesystem plugin [#4421] [#6557]
+ - Add connection_pool features [#5060] [#5799] [#7130] [#7145] [#7287] [#7304] [#7314] [#7350]
+ - Add ticket administration library [#5716] [#7115] [#7179] [#7185] [#7211]
+ - Add server configuration reload on SIGHUP [#5816] [#5821] [#6569]
+ - Add client connection information to acPreConnect [#5985]
+ - Use boost::process to capture messages better [#6273]
+ - Expose CRON task intervals in advanced settings [#6342]
+ - Expose delay server and control plane timeout settings [#6370]
+ - Improved logging facilities [#6470] [#6493] [#6641] [#6642] [#6818] [#7048] [#7212]
+ - Add remote configuration reload to irods-grid [#6529]
+ - Add interactive_pam authentication plugin [#6576]
```

Comment:
Not sure if it helps to clarify things or not, but we did not add a new plugin - just added support for it to `iinit`.

Reply:
ah - noted.  thanks.